Makefile

```
@@ -127,6 +127,18 @@ test-go: ## Run tests for backend.
 	@echo "test backend"
 	$(GO) test -v ./pkg/...
 
+test-go-unit: ## Run unit tests for backend.
```

I would suggest changing the comments to emphasise how this differs from the `test-go` target.
```suggestion
test-go-unit: ## Run unit tests for backend with flags.
```

---

I added "with flags" to the end of the new makefile commands I created - how does it look now?